src/services/services.ts

```diff
@@ -4694,6 +4694,10 @@ namespace ts {
                         if (!(container.kind === SyntaxKind.ModuleBlock || container.kind === SyntaxKind.SourceFile)) {
                             return undefined;
                         }
+                    } else if (modifier === SyntaxKind.AbstractKeyword) {
+                        if (!(container.kind === SyntaxKind.ClassDeclaration || declaration.kind === SyntaxKind.ClassDeclaration)) {

```

Comment:
Don't think these are the same or am I totally wrong here? On members of a class the container.kind is the ClassDeclaration, on the abstract keyword of the class the container is actually a source file so that is why I added the declaration.kind check.

Will do some test for ClassExpressions. Thanks for the feedback!